src/routes/index.tsx

```diff
@@ -58,12 +58,12 @@ const Routes: React.FC = () => {
 		<NavigationContainer>
 			<Stack.Navigator initialRouteName={initialScreen} screenOptions={{ headerShown: false }}>
 				<Stack.Screen name="Welcome" component={WelcomeScreen} />
-				<Stack.Screen name="Assets" component={AssetsScreen} />
+				<Stack.Screen name="AssetsScreen" component={AssetsScreen} />
```

Comment:
looks like we don't use the word Screen after the screen name here. Can we follow this pattern or change the others to include screen?

Reply:
or maybe I did not understand it properly.